view/gui.py

```diff
@@ -0,0 +1,46 @@
+import streamlit as st
+import folium
+import osmnx
+import networkx as nx
+import leafmap.foliumap as leafmap
+import json
+
+class gui:
+    def __init__(self, filename = "config.json"):
+        with open(filename) as config_file:
+            self.config = json.load(config_file)
+    
+    def clear_text(self):
+        st.session_state["go_from"] = ""
+        st.session_state["go_to"] = ""
+
+    def generate_and_show_gui(self):
```

Comment:
It seems to be missing the initial map being displayed. For now, set the longitude latitude of the map to be 


```
lat, lon = 42.4529, -72.5653

m = leafmap.Map(center=(lat, lon), zoom=16)

m.add_basemap(basemap)

m.to_streamlit()
```

Reply:
Done